### PR TITLE
Add 'User settings' to application-settings

### LINKED ISSUE #1673: Migrate Clipboard export user settings to Application settings

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #1673

We decided to have the following categories as separate section and not bundled:
- Theme settings
- Display Language
- Clipboard export format

### CODE REVIEW

**Comment by ollimeier** [comment]
@justvanrossum This is how it looks like:
Screenshot 2024-09-25 at 15 49 03

File: src/fontra/views/editor/editor.js [hunk 1]
```diff
@@ -128,8 +129,7 @@ export class EditorController {
       async (...args) => await this.editListenerCallback(...args)
     );
 
-    this.clipboardFormatController = new ObservableController({ format: "glif" });
-    this.clipboardFormatController.synchronizeWithLocalStorage("fontra-clipboard-");
+    this.clipboardFormatController = clipboardFormatController;
```
Comment (justvanrossum) [inline src/fontra/views/editor/editor.js:132]: Suggest to just use `clipboardFormatController` instead of `this.clipboardFormatController`.

**Review by justvanrossum — COMMENTED** [review]
I'm not super happy with all the "card" repetitions in the panel code and CSS: let's create a follow-up issue to fix that.

Other than that, one small comment, the rest is fine.

File: src/fontra/views/editor/editor.js (hunk 1 above)
Comment (ollimeier) [reply src/fontra/views/editor/editor.js:132]: But will the change not break other parts?

Comment (justvanrossum) [reply src/fontra/views/editor/editor.js:132]: > But will the change not break other parts?

You can do a global search for `.clipboardFormatController`. It looks like it's used by the gear panel and editor.js, and nothing else.

Comment (ollimeier) [reply src/fontra/views/editor/editor.js]: ok, done.

**Review by justvanrossum — COMMENTED** [review]
LGTM, except I'd like the "Server info" panel to be the last in both the menu and the panel items sidebar, as it's the least important one.

File: src/fontra/views/applicationsettings/applicationsettings.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <link href="/css/tooltip.css" rel="stylesheet" />
     <title>Fontra Application Settings</title>
     <script type="module" src="/web-components/modal-dialog.js"></script>
+    <script type="module" src="/web-components/grouped-settings.js"></script>
```
Comment (justvanrossum) [inline src/fontra/views/applicationsettings/applicationsettings.html:9]: Please add
```
<script type="module" src="/core/theme-settings.js"></script>
```
after this line, as per #1650.

(While it works without this, given the inclusion of the theme panel, it is good practice and a necessary example for other views.)

Comment (ollimeier) [reply src/fontra/views/applicationsettings/applicationsettings.html:9]: ok, done.

**Pull request merged**